EIPS/eip-695.md

```diff
@@ -36,6 +36,10 @@ the RPC.
 Returns the currently configured chain ID, a value used in replay-protected transaction
 signing as introduced by [EIP-155](./eip-155.md).
```

Comment:
All these links should be pointing to `https://eips.ethereum.org/EIPS/eip-155, etc.